Makefile

```diff
@@ -7,7 +7,7 @@ CFLAGS+=-I$(INCDIR)
 SRCS=$(wildcard $(SRCDIR)/*.c)
 OBJS=$(patsubst $(SRCDIR)/%.c,$(OBJDIR)/%.o,$(SRCS))
 
-CFLAGS+=-O2 -Wall -std=c99
+CFLAGS+=-O2 -Wall -std=c99 `pkg-config --cflags --libs gtk+-3.0`
```

Comment:
It seems incorrect anyway. Don't you really want `pkg-config --cflags --libs gdk-pixbuf-2.0`? gdk-pixbuf is a separate package for exactly this reason AFAIK.

(I'm not sure exactly what is going on with GDKPixbuf in relation to gtk3, but I'm pretty sure we don't need to even try to link gtk3 to get the library we actually want here)